wikibase/1.30/bundle/Dockerfile

```diff
@@ -3,10 +3,22 @@ FROM ubuntu as fetcher
 RUN apt-get update && \
     apt-get install --yes --no-install-recommends unzip jq curl ca-certificates
 ADD download-extension.sh .
-RUN bash download-extension.sh OAuth
-RUN tar xzf OAuth.tar.gz && rm *.tar.gz
+RUN bash download-extension.sh OAuth;\
+bash download-extension.sh Elastica;\
+bash download-extension.sh CirrusSearch;\
+tar xzf OAuth.tar.gz;\
+tar xzf Elastica.tar.gz;\
+tar xzf CirrusSearch.tar.gz
```

Comment:
I compressed them together; this is what it looks like now.

IMO every so slightly less readable with the messy ;\ at the line ends.

Let's not bike shed on this though :P. I'll be content with either